code/game/turfs/simulated.dm

```diff
@@ -54,8 +54,8 @@
 /turf/simulated/Entered(atom/A, atom/OL, ignoreRest = 0)
 	..()
 	if(!ignoreRest)
-		if(ismob(A)) //only mobs make dirt
-			if(prob(80))
+		if(ismob(A) && isliving(A))

```

Comment:
Why would a non-corporeal ghost that can only make lights flicker create physical muck?